CK2Plus/common/scripted_triggers/000_CK2Plus_title_requirements.txt

```diff
@@ -0,0 +1,64 @@
+#########################################
+#
+#
+# Title Requirement Triggers
+#
+#
+#########################################
+
+#These MUST be called in a CHARACTER scope to work properly
+
+has_ck2plus_duchy_reqs = {
+	# If title is titular
+	conditional_tooltip = {
+		trigger = { FROM = { is_titular = yes } }
+		lower_real_tier_than = DUKE
+	}
+	# If title is landed
+	conditional_tooltip = {
+		trigger = { FROM = { is_landed = yes } }
+		# Alternate Reqs to go here later
+	}
+	# Apply regardless
+	is_adult = yes
+	has_regent = no
+	war = no
+}
+
+has_ck2plus_kingdom_reqs = {
+	# If title is titular
+	conditional_tooltip = {
+		trigger = { FROM = { is_titular = yes } }
+		lower_real_tier_than = KING
+	}
+	# If title is landed
+	conditional_tooltip = {
+		trigger = { FROM = { is_landed = yes } }
+		# Alternate Reqs to go here later
+	}
+	# Apply Regardless
+	is_title_active = FROM
+	is_adult = yes
+	has_regent = no
+	war = no
+	ruled_years = 5
+}
+
+has_ck2plus_empire_reqs = {
+	# If title is titular
+	conditional_tooltip = {
+		trigger = { FROM = { is_titular = yes } }
+		lower_real_tier_than = EMPEROR
+	}
+	# If title is landed
+	conditional_tooltip = {
+		trigger = { FROM = { is_landed = yes } }
+		# Alternate Reqs to go here later
```

Comment:
Could we make comments like this in the format `# TODO: The thing` so that they are easier to search for later?

Reply:
Can do. I'll get that edited in a later commit since this one has already been merged